common/tier4_localization_rviz_plugin/src/pose_history_footprint/display.hpp

```diff
@@ -59,17 +58,17 @@ class PoseHistoryFootprint
   void onInitialize() override;
   void onEnable() override;
   void onDisable() override;
-  void updateFootprint();
+  void update_footprint();
 
 private Q_SLOTS:
-  void updateVisualization();
-  void updateVehicleInfo();
+  void update_visualization();
+  void update_vehicle_info();
 
-private:
+private:  // NOLINT
```

Comment:
Please add the comment why need `NOLINT`

Reply:
I added  comments for `NOLINT` lines in de72c7d .